resume_agent/backend/resume_agents.py

```
@@ -0,0 +1,204 @@
+
+    def __init__(self, api_key: Optional[str] = None, model: str = "gpt-4o-mini"):
+        """
+        Initialize the resume ranking agent.
+
+        Args:
+            api_key: OpenAI API key (defaults to environment variable)
+            model: LLM model to use
+        """
+        self.api_key = api_key or os.getenv("OPENAI_API_KEY")
+        if not self.api_key:
+            raise ValueError("OpenAI API key is required")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)